models/gmx/arbitrum/gmx_arbitrum_schema.yml

```diff
@@ -115,7 +111,7 @@ models:
             - version
             - tx_hash
             - evt_index
-      - check_perpetuals_seed_legacy:
+      - check_perpetuals_seed:
```

Comment:
assuming this was a universal find/replace -- i think you should actually leave perpetual as legacy, since that is another sector of spells completely